noir-projects/aztec-nr/aztec/src/state_vars/private_immutable.nr

```diff
@@ -137,36 +83,15 @@ impl<Note> OwnedPrivateImmutable<Note, &mut PrivateContext> {
         self.compute_initialization_nullifier(secret)
     }
 
```

Comment:
Pardon me for I have sinned and unnecessarily increased the scope of this PR. As once discussed in scrum the rigid style of the docs below is just too verbose and the `Advanced` section is describing implementation which is unhelpful.

Since I was already touching the docs decided to make it more concise and formatted it to 120 char line width.